CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## Unreleased
 
+## [1.1.4] 2019-01-09
```

Comment:
In general the versioning and dating are left to the maintainers for several reasons:
- we may disagree on the impact of the change and therefore how its versioned (in this case it would be a major version as dropping ruby support and bumping the `sensu-plugin` dep past a major are both breaking changes)
- we date the release rather then when the contribution was submitted, PR review and acceptance has no SLA and is not guaranteed to be merged within the same day
- PRs are not a FIFO queue, as such versioning them prior to acceptance leads to needing to ask people to rebase more often

Reply:
Noted; I am removing this header so that it can be set when a release is cut.